snackager/package.json

```diff
@@ -82,7 +82,7 @@
     "lint": "eslint ./src"
   },
   "volta": {
-    "node": "12.18.4",
+    "node": "16",
```

Comment:
I would put a slightly more specific version here for volta, [like the current latest 16 version](https://nodejs.org/download/release/latest-v16.x/): `16.14.2`